### DISCUSSION TITLE
Is automation of Cloud Foundry setup for a new global account not possible?

### DISCUSSION THREAD

**Discussion by dxs-dbest** [SAP-archive/btp-setup-automator#228] [Q&A] [answered]
My goal with this tool was to be able to automate the configuration of new global accounts (we are an SAP partner who hosts our application on BTP for our customers).  I have tried the default usecase and it creates services/subscriptions fine and it creates global/subaccount users fine. It leaves me with an org with one member and a space with one member (my ID).  

Any cf commands trying to set org or space members end up looking like this if I run them separate from the btpsa script:

```
0f6bb61c7447:~$ cf set-space-role 'john.admin@test.com' 'cf-bptsa-66a7201b-9ad8-4dbc-8e9f-f288bc700f90' 'development' 'SpaceDeveloper'
Assigning role SpaceDeveloper to user john.admin@test.com in org cf-bptsa-66a7201b-9ad8-4dbc-8e9f-f288bc700f90 / space development as xxxxxxxx@xxxxxxxx.com...
No user exists with the username 'john.admin@test.com'.
FAILED
```

This [SAP documentation](https://help.sap.com/docs/BTP/65de2977205c403bbc107264b8eccf4b/1422a5daa53d498f9270727135005884.html) seems to say that I won't be able to automate the setup of the CF org and space because I don't want to create the user in the cockpit and won't be able to have users log in with cf login before setting up the org/space.  Is my understanding correct?  I am using the parameters.json file in the docker container root.  Should I expect to see tony@stark.com as a member of the space with SpaceManager or will the script only work if those are just placeholders that get replaced with real users?  

Thanks,
David

**Comment by lechnerc77** [comment, marked as answer]
Hi @dxs-dbest, 
I think you should be able to achieve that. If I understand you correctly one fitting example is the use case file [cap_app_launchpad.json](https://github.com/SAP-samples/btp-setup-automator/blob/main/usecases/released/cap_app_launchpad.json). where several roles are assigned e. g. an org admin.
You need to define the corresponding user and groups in the `parameters.json` file in the `myusergroups` section. The default file gives you an idea how this should like like.  

This discussion ([link](https://github.com/SAP-samples/btp-setup-automator/discussions/119#discussion-4072131)) might also give you helpful insights.

Let me know if this is what you are looking for.

**Reply by lechnerc77** [reply]
No feedback since > one month => mark question as answered
